### ISSUE TITLE
Rule Proposal: Correct chaining

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#36]
From time to time, I see the incorrect element, element.all chaining like in [this topic](http://stackoverflow.com/questions/38399980/protractor-repeater-inside-element/38399999#38399999).

Invalid usage:

``` js
var dates = $('#provider_details_view').element.all(by.repeater('repeater name');
```

Valid usage:

``` js
var dates = $('#provider_details_view').all(by.repeater('repeater name');
```

**Comment by alecxe** [comment]
Closed by bb1c9a8411e42f01e48adce6ec822e2375fa4aac.
